src/plugins/api_tests/src/core/external_script/base_scheme_interface/CInterfaceSasTests.cpp

```diff
@@ -31,12 +32,32 @@
 
 #include "SchemeSimilarityUtils.h"
 
-static const QString WORKING_DIR = U2::AppContext::getWorkingDirectoryPath();
-#ifndef Q_OS_MAC
-static const QString PROPER_WD_SCHEMES_PATH = WORKING_DIR + "/../../test/_common_data/cmdline/wd-sas-schemes/";
-#else
-static const QString PROPER_WD_SCHEMES_PATH = WORKING_DIR + "/../../../../../../test/_common_data/cmdline/wd-sas-schemes/";
-#endif
+#include <QFileInfo>
+
+static QString getTestDirImpl() {
+    QString testDir = qgetenv("UGENE_TESTS_PATH");
+    auto getDefaultTestDir = []() -> QString {
+        QString defaultTestDir;
+        #ifndef Q_OS_MAC
+        defaultTestDir = U2::AppContext::getWorkingDirectoryPath() + "/../../test/";
+        #else
+        defaultTestDir = U2::AppContext::getWorkingDirectoryPath() + "/../../../../../../test/";
+        #endif
+        return defaultTestDir;
+    };
+    if (testDir.isEmpty()) {
+        testDir = getDefaultTestDir();
+    } else if (!QFileInfo::exists(testDir)) {
```

Comment:
Is it possible to create a directory if it doesn't exist? Or doesn't it make sense?

Reply:
The "UGENE_TESTS_PATH" using supposes that this directory already exists